working/augmentation-libraries/feature-specification.md

```diff
@@ -301,7 +299,7 @@ It is a compile-time error if:
 
 *   An augmenting declaration appears in a library before the library where the
     original declaration occurs, according to merge order. *An augmentation
-    library can both declare a new declaration and augment it in the same file.*
+    can both declare a new declaration and augment it in the same file.*
```

Comment:
(Can it augment the same member twice in the same `augment class`? ... I know, I should just check. Should it?)

Reply:
Yes it can, and we do this in the macro generated augmentations to make them more concise, if multiple macros augment the same member